brambling/views/organizer.py

```diff
@@ -1249,10 +1299,12 @@ def get_forms(self):
                         form.data = self.request.POST
                         form.is_bound = True
                         break
+        self.transaction_forms = [TransactionRefundForm(t)
+                                  for t in self.order.transactions.all()]
         if show_payment_form:
```

Comment:
Uhhhhhh so we're not actually hiding _any_ of the forms. The `forms` var here gets returned in this method, but it doesn't actually get used anywhere. The forms are stored on the view instance and are _always_ included in template context. But the template checks for edit permissions before displaying the forms. I'll take the same approach to transaction refund forms now, but I wonder if they should never be passed to the context at all if there aren't permissions...